### PR TITLE
Color UUIDs in yellow & Format time for statistics

### PR DESCRIPTION
**Pull request by MatthewKhouzam**
Signed-off-by: Matthew Khouzam <matthew.khouzam@ericsson.com>

### CODE REVIEW

File: tsp-cli-client [hunk 1]
```diff
@@ -26,9 +26,17 @@ import argparse
...
 
 #api_token = 'your_token_goes_here'
 
+
+def printUuid(elem):
+    return colored(elem.UUID, 'yellow')
+
+def printTimestamp(elem):
+    return     
+
```
Comment (MatthewKhouzam) [inline tsp-cli-client]: remove

**Comment by bhufmann** [comment]
Please add termcolor to the requirements.txt file. Otherwise it looks good.
Thanks
Bernd

**Comment by MatthewKhouzam** [comment]
thanks!

**Comment by bhufmann** [comment]
Thanks Matthew!
It looks good

**Pull request merged**